templates/CRM/Core/DatePickerRangeWrapper.tpl

```diff
@@ -25,5 +25,6 @@
 *}
 {* Wrapper around DatePickerRange TPL file *}
 <td {if $colspan} colspan="{$colspan}" {else} colspan="2" {/if} {if $class} class="{$class}" {/if}>
-    {include file="CRM/Core/DatePickerRange.tpl" fieldName=$fieldName}
+  {assign var='hideRelativeLabel' value=$hideRelativeLabel|default:0}
```

Comment:
I can't see which field we are hiding relative for?

Reply:
The date ones because in the current initiation of the contribution recur sub forms the labels are displayed in the first column with the field content in the 2nd column rather than label above the field content

Reply:
hmm - I feel like it would make  sense to stdise the UI for those recur dates to be the same as the others - even though I prefer the label not to be above it is on all the other sub-tabs & stdisation feels like it would help themers etc as well as reduce extra code handling

Reply:
I'm still with changing the layout to match other tabs here

Reply:
Ok - I'm softening on this - if we merge #14825 & get this rebased  & & fix the miscasing above it should be pretty close